regression/esbmc/github_2338_7/main.c

```diff
@@ -0,0 +1,8 @@
+#include <assert.h>
+
+int main() {
+  unsigned int a = 0x80000001;
+  unsigned int result = -a;
+  assert(result == 0x7FFFFFFF); // Expected modular negation
```

Comment:
I think this assert is never actually triggered, because the negation will cause an error beforehand.

Reply:
Assertion removed.